packages/manager/apps/dedicated/client/app/account/user/user.routes.js

```diff
@@ -28,6 +29,23 @@ export default /* @ngInject */ ($stateProvider) => {
           : Promise.resolve(null),
       breadcrumb: /* @ngInject */ ($translate) =>
         $translate.instant('user_account'),
+
+      guides: /* @ngInject */ (currentUser) => {
+        Object.entries(GUIDES_LIST).forEach(([key, value]) => {
+          GUIDES_LIST[key].url =
+            typeof value.url === 'object'
```

Comment:
question, i think it could be a mistake for one case.
if url is null so the typeof will return `'object'` because (`typeof null = 'object'`), and the true statement will be executed.

may be it's better to try with this:
```suggestion
            const subUrl = value?.url?.[currentUser.ovhSubsidiary] ;
            GUIDES_LIST[key].url = subUrl ? subUrl : value.url;
```

Reply:
I changed my approach. Knowing that url cannot be null because it is a constant not an api.